packages/server-wallet/src/models/channel.ts

```diff
@@ -418,6 +421,12 @@ export class Channel extends Model implements RequiredColumns {
   }
 
   private get _support(): Array<SignedStateWithHash> {
+    // TODO: activate these fields for proper application checks (may be resource hungry)
```

Comment:
Should this todo be done as part of this work?

Reply:
I think not, as that would alter the behaviour of the code (not my intention, here). It does highlight the fact that we do not currently check the application-level valid transition function, though. We should probably have an issue to help track our journey getting to that point.

Reply:
For extra context, this comment merely replaces

```
  // We will eventually want to perform a proper validTransition check	
  // but we will have to be careful where we do that to prevent eating up a ton of cpu
```